docs/installation_and_tutorial.md

```diff
@@ -343,32 +260,43 @@ Finally the logs are shown in the runtime:
 
 ## Trends
 
-To add a trend go to the add screen items and in the advanced bar select trend control:
+To add a trend go to the Toolbox, select trend control and drag and drop into the screen:
 
-![trends1](graphics/trends1_new.jpg)
+![trends1](graphics/trends1.png)
 
-In its properties to add different trends to appear in the item, go to Trends:
+In its properties to add different trends to appear in the item, go to Trends and select the tag/logging tag you want to control::
 
 ![trends2](graphics/trends2.png)
 
-And select the tag you want to control:
-
-![trends3](graphics/trends3.png)
-
 In the runtime the trend will be filled:
 
 ![trends4](graphics/trends4.png)
 
-## Start the runtime
+## TIA Portal guidelines
+
+In case you need more information related to the engineering of the project in TIA Portal, you can refer to the following [documentation](https://cache.industry.siemens.com/dl/dl-media/433/109782433/att_1265883/v1/109782433_SIMATIC-WinCC-Unified-Tutorial-Center_140324_web/start.htm#!/en).
+
+## How to use WinCC Unified Runtime Manager
 
 To start the runtime go to the Wincc Runtime app:
 
 ![start1](graphics/start1.png)
 
+You will see a login page, where you can login with the credentials downloaded within the project. At least one user with administration rights is required to access the Web Runtime Manager.
+<br>Note: Default user: **uoeuser**, Default password: **User@uoe**
+
+![login](graphics/login.png)
+
+If you want to add a user to change the access rights or any other reason, you can do it in the TIA Project:
+
```

Comment:
We should also add a hint, that the default-user uoeuser will be overwritten once a TIA Portal project will be downloaded with UMC-users